docs/AutocompleteInput.md

```diff
@@ -6,54 +6,69 @@ title: "The AutocompleteInput Component"
 # `<AutocompleteInput>`
 
 To let users choose a value in a list using a dropdown with autocompletion, use `<AutocompleteInput>`.
-It renders using MUI [Autocomplete](https://mui.com/components/autocomplete/).
+It renders using [MUI's `<Autocomplete>`](https://mui.com/components/autocomplete/).
 
 ![AutocompleteInput](./img/autocomplete-input.gif)
 
-Set the `choices` attribute to determine the options list. The value must be an array of objects exposing the `id` and `name` for each choice.
+This input allows editing record fields that are scalar values, e.g. `123`, `'admin'`, etc. 
+
+## Usage
+
+In addition to the `source`, `<AutocompleteInput>` requires one prop: the `choices` listing the possible values.
```

Comment:
Actually, `choices` is optional, and we say that in the props table below

Reply:
In the general case, `choices` is required. I detail the general case in the Usage section, then the specific cases in each prop description. For simplicity's sake, I prefer to leave it like that.